velox/docs/functions/spark/math.rst

```diff
@@ -216,3 +216,8 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: isnan(x) -> boolean
```

Comment:
Make sure functions are listed in alphabetical order.

Reply:
Updated, thanks.